standard-controller/src/main/java/io/enmasse/controller/standard/TemplateBrokerSetGenerator.java

```diff
@@ -110,6 +110,11 @@ private KubernetesList processTemplate(String clusterId, int numReplicas, Addres
             if (item instanceof StatefulSet) {
                 StatefulSet set = (StatefulSet) item;
                 set.getSpec().setReplicas(numReplicas);
+                if (standardInfraConfig.getSpec().getBroker().getStorageClassName() != null) {
```

Comment:
The Broker object within the StandardInfraConfig could be absent, so a null check is required for `standardInfraConfig.getSpec().getBroker()`